firefox-ios/Client/Frontend/Browser/Tabs/Views/ExperimentEmptyPrivateTabsView.swift

```diff
@@ -8,10 +8,9 @@ import Foundation
 import Shared
 import ComponentLibrary
 
+@MainActor
```

Comment:
Just a quick question: how should we decide when to annotate an entire protocol with @MainActor versus adding it only to individual methods or conforming types (like in AddressToolbarDelegate above)?